app/src/test/java/org/fossasia/openevent/app/unit/presenter/AttendeeCheckInPresenterTest.java

```diff
@@ -124,7 +125,7 @@ public void shouldHandleAttendeeChange() {
         notifier.onNext(new DatabaseChangeListener.ModelChange<>(toggled, BaseModel.Action.UPDATE));
 
         verify(attendeeCheckInView).showResult(toggled);
-        verify(attendeeCheckInView).onSuccess(any());
+        verify(attendeeCheckInView, atLeast(2)).onSuccess(any());
```

Comment:
>But why will it be called twice on a single change?

not single
1. first checking is set to true and checkedIn is toggled
2. and then checking is reset to false.